lib/localize.js

```diff
@@ -164,8 +191,15 @@ export const getLocalizeClass = (superclass = class {}) => class LocalizeClass e
 		}
 		if (Object.prototype.hasOwnProperty.call(this, 'getLocalizeResources') || Object.prototype.hasOwnProperty.call(this, 'resources')) {
 			const possibleLanguages = this._generatePossibleLanguages(config);
-			const resourcesPromise = this.getLocalizeResources(possibleLanguages, config);
-			resourcesLoadedPromises.push(resourcesPromise);
+			if (config?.importFunc) {
```

Comment:
I had to add this check to handle the case where the consumer _only_ wants to use the common resources.